config/ConfigureChecks.cmake

```diff
@@ -610,17 +610,22 @@ endif ()
 #  Check if ROS3 driver can be built
 #-----------------------------------------------------------------------------
 option (HDF5_ENABLE_ROS3_VFD "Build the ROS3 Virtual File Driver" OFF)
-  if (HDF5_ENABLE_ROS3_VFD)
-    find_package(CURL REQUIRED)
-    find_package(OpenSSL REQUIRED)
-    if (${CURL_FOUND} AND ${OPENSSL_FOUND})
-      set (${HDF_PREFIX}_HAVE_ROS3_VFD 1)
-      list (APPEND LINK_LIBS ${CURL_LIBRARIES} ${OPENSSL_LIBRARIES})
-      INCLUDE_DIRECTORIES (${CURL_INCLUDE_DIRS} ${OPENSSL_INCLUDE_DIR})
-    else ()
-      set (HDF5_ENABLE_ROS3_VFD OFF CACHE BOOL "Build the ROS3 Virtual File Driver" FORCE)
-      message (WARNING "The Read-Only S3 VFD was requested but cannot be built.\nPlease check that openssl and cURL are available on your\nsystem, and/or re-configure without option HDF5_ENABLE_ROS3_VFD.")
+if (HDF5_ENABLE_ROS3_VFD)
+  # The ROS3 VFD requires the aws-c-s3 library
+  find_package (aws-c-s3 REQUIRED)
```

Comment:
Since CMake does not have a Find Module for aws maybe specify CONFIG

Reply:
Happy to change it, though maybe in a followup PR to avoid dismissing reviews since it's such a minor change?